sound/soc/sof/amd/acp.c

```diff
@@ -390,6 +390,7 @@ static int acp_power_on(struct snd_sof_dev *sdev)
 
 static int acp_reset(struct snd_sof_dev *sdev)
 {
+	const struct sof_amd_acp_desc *desc = get_chip_info(sdev->pdata);
```

Comment:
please fix typos in the commit message "select" "selected as" , "internal clock to 96MHz" -> "Internal clock of...", "depends on usacase" -> "depending on usecase".

Reply:
Thank you